webapp/app/services/roleService.js

```diff
@@ -1,17 +1,43 @@
 define('services/roleService', [], function () {
 
     function roleReadAll() {
-    // TODO: refactor common headers and base url
-    return $.ajax({
-      type: 'POST',
-      contentType: 'application/json',
-      url:'http://localhost:10160/Service.svc/jsonp/post',
-      data: '{"requests":[{"__type":"RoleReadAllRequest:#P202.Training.WCF.RequestsAndResponses"}]}'
-    });
-  }
+        console.log('Entre a llamar WebService')
+        return $.ajax({
+            type: 'POST',
+            contentType: 'application/json',
+            url: 'http://localhost:10160/Service.svc/jsonp/post',
+            data: '{"requests":[{"__type":"RoleReadAllRequest:#P202.Training.WCF.RequestsAndResponses"}]}'
+        });
+    }
 
-  return {
-      roleReadAll: roleReadAll
-  };
+    function roleAdd(value) {
+        var roleIns = {
+            "Name":value
+        };              
+        var roleInsJs = JSON.stringify(roleIns);
+        return $.ajax({
+            type: 'POST',
+            contentType: 'application/json',
+            url: 'http://localhost:10160/Service.svc/jsonp/post',
+            data: '{"requests":[{"__type":"RoleAddRequest:#P202.Training.WCF.RequestsAndResponses","Role":' + roleInsJs + '}]}'
+        });
+    }
+
+    function roleDelete(value) {
+        // TODO: refactor common headers and base url
+        console.log('roleServices.js/rokeDelete');
```

Comment:
Minor: please remove console.log lines before merging

Reply:
i remove the console.log